Testreglar/134/App/app-1.3.4a.json

```diff
@@ -70,25 +72,55 @@
 		},
 		{
 			"stegnr": "3.3",
-			"spm": "Finnes det en fungerende mekanisme for å endre visningsretningen?",
-			"ht": "<p>Gjør en visuell inspeksjon, og se etter om det finnes en mekanisme som lar brukeren endre visningsretning. Eksempel på en mekanisme kan være en knapp.</p>\n<p>Aktiver mekanismen og sjekk om visningsretningen for innholdet blir endret.</p>",
+			"spm": "Kan alt innhold på appsiden brukes i begge visningsretninger?",
+			"ht": "<p>Gjør visuell inspeksjon og sjekk om alt innhold kan brukes i begge visningsretninger.</p>",
 			"type": "jaNei",
-			"kilde": [
-				"G214"
-			],
+			"kilde": [],
 			"ruting": {
 				"ja": {
 					"type": "avslutt",
 					"fasit": "Ja",
-					"utfall": "Det finnes en mekanisme for å endre visningsretning."
+					"utfall": "Det er mulig å vise og bruke innholdet både i stående og liggende retning."
 				},
 				"nei": {
+					"type": "gaaTil",
+					"steg": "3.4"
+				}
+			}
+		},
+		{
+			"stegnr": "3.4",
+			"spm": "Beskriv element eller funksjonalitet du ikke fikk til å bruke.",
+			"ht": "<p>Beskriv elementet eller funksjonaliteten. Hvis det gjelder flere elementer eller funksjonalitet, registrerer du en og en.</p>",
+			"type": "tekst",
+			"kilde": [],
+			"label": "Element:",
+			"multilinje": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.5"
+				}
+			}
+		},
+		{
+			"stegnr": "3.5",
+			"spm": "I hvilken visningsretning fikk du ikke til å bruke elementet eller funksjonaliteten?",
+			"ht": "",
+			"type": "radio",
+			"kilde": [],
+			"svarArray": [
+				"Stående retning",
+				"Liggende retning"
+			],
+			"ruting": {
+				"alle": {
 					"type": "avslutt",
 					"fasit": "Nei",
-					"utfall": "Det er ikke mulig å vise innholdet både i stående og liggende retning."
+					"utfall": "Det er ikke mulig å vise og bruke innholdet både i stående og liggende retning."
```

Comment:
samme utfall for begge valg. Usikker på om det gir noe merverdi å registrere hvilken retning de feiler på. Bør i såfall ha ett utfall per valg? Henger sammen med om man kun skal registrere brudd ref tidligere kommentar

Reply:
Ja, det er et ekstra steg for å registrere data om i hvilken retning feilen ligger. Hvis det er unødvendig så kan vi fjerne den, det er kun for å vite om det i etterkant når brudd skal rapporteres. Meningen er ikke å gjøre denne testregelen for kompleks.

Reply:
Kunne det ene utfallet være " Det er ikke mulig å vise alt innhold i liggende retning" og det andre "Det er ikke mulig å vise alt innhold i stående retning"?

Reply:
Ja, men ikke mulig her. Dette er et ekstra steg for å registere data om visnings retning.